filebeat/module/elasticsearch/slowlog/test/es_index_indexing_slowlog-json.log-expected.json

```diff
@@ -16,7 +16,7 @@
         "event.dataset": "elasticsearch.slowlog",
         "event.duration": 4000000,
         "event.module": "elasticsearch",
-        "event.timezone": "+00:00",
+        "event.timezone": "-02:00",
```

Comment:
Similar to my other comment: https://github.com/elastic/beats/pull/13874/files#r330526232.